packages/cli/src/commands/org/config/pull.ts

```diff
@@ -0,0 +1,62 @@
+import {ResourceType} from '@coveord/platform-client';
+import {flags, Command} from '@oclif/command';
+import {Config} from '../../../lib/config/config';
+import {
+  IsAuthenticated,
+  Preconditions,
+} from '../../../lib/decorators/preconditions';
+import {Snapshot} from '../../../lib/snapshot/snapshot';
+import {SnapshotFactory} from '../../../lib/snapshot/snapshotFactory';
+
+export default class Pull extends Command {
+  public static description = 'Pull resources from an organization';
+
+  public static flags = {
+    target: flags.string({
+      char: 't',
+      description:
+        'The unique identifier of the organization from which to pull the resources. If not specified, the organization you are connected to will be used.',
+      helpValue: 'destinationorganizationg7dg3gd',
+      required: false,
+    }),
+    // TODO: CDX-447: add flag to select resource types to export
```

Comment:
Question: How will we keep that list of ressources up to date ? If we hardcode them in the CLI, then it means whenever a new type of resource is added in the platform, we will be lagging behind.

Which is annoying for end users, and for us.

Any way to do this in an intelligent manner ? 🤔

Reply:
Don't worry. It will not be harcoded. 
All the available resource types will be directly coming from platform-client.